pages/services/cassandra/2.6.0-3.11.4/operations/index.md

```diff
@@ -0,0 +1,313 @@
+```
+dcos {{ model.packageName }} --name=<service-name> plan status repair
+repair (STARTING)
+└─ repair-deploy (STARTING)
+   ├─ node-0:[repair] (STARTING)
+   ├─ node-1:[repair] (PENDING)
+   └─ node-2:[repair] (PENDING)
+```
+
+When the plan is completed, its status will be `COMPLETE`.
+
+The above `plan start` and `plan status` commands may also be made directly to the service over HTTP. To see the queries involved, run the above commands with an additional `-v` flag.
+
+For more information about `nodetool repair`, see the {{ model.techShortName }} documentation.
+
+## Seed nodes
+
+{{ model.techShortName }} seed nodes are those nodes with indices smaller than the seed node count.  By default, {{ model.techShortName }} is deployed
+with a seed node count of two (node-0 and node-1 are seed nodes). When a replace operation is performed on one these
+nodes, all other nodes must be restarted to be brought up to date regarding the ip address of the new seed node. This
+operation is performed automatically.
+
+For example if `node-0` needed to be replaced you would execute:
+
+```bash
+dcos {{ model.packageName }} --name=<service-name> pod replace node-0
+```
+
+which would result in a recovery plan like the following:
+
+```bash
+$ dcos {{ model.packageName }} --name=<service-name> plan show recovery
+recovery (IN_PROGRESS)
+└─ permanent-node-failure-recovery (IN_PROGRESS)
+   ├─ node-0:[server] (COMPLETE)
+   ├─ node-1:[server] (STARTING)
+   └─ node-2:[server] (PENDING)
+   ...
+```
+
+<p class="message--note"><strong>NOTE: </strong>Only the seed node is being placed on a new node, all other nodes are restarted in place with no loss of data.</p>
+
+
+# Back up and Restore
+
+## Backing Up to S3
+
+You can back up an entire cluster's data and schema to Amazon S3 using the `backup-s3` plan. This plan requires the following parameters to run:
+- `SNAPSHOT_NAME`: the name of this snapshot. Snapshots for individual nodes will be stored as S3 folders inside of a top level `snapshot` folder.
+- `CASSANDRA_KEYSPACES`: the {{ model.techShortName }} keyspaces to back up. The entire keyspace, as well as its schema, will be backed up for each keyspace specified.
+- `AWS_ACCESS_KEY_ID`: the access key ID for the AWS IAM user running this backup
+- `AWS_SECRET_ACCESS_KEY`: the secret access key for the AWS IAM user running this backup
+- `AWS_REGION`: the region of the S3 bucket being used to store this backup
+- `S3_BUCKET_NAME`: the name of the S3 bucket in which to store this backup
+- `HTTPS_PROXY`: specifications for the {{ model.TechName }} backup plan, taken from `config.yaml`.
+
+Make sure that you provision your nodes with enough disk space to perform a backup. {{ model.TechName }} backups are stored on disk before being uploaded to S3, and will take up as much space as the data currently in the tables, so you will need half of your total available space to be free to back up every keyspace at once.
```

Comment:
```suggestion
Make sure to provision your nodes with enough disk space to perform a backup. {{ model.TechName }} backups are stored on disk before being uploaded to S3. It will take up more space as the data currently in the tables and requires half of your total available space to be free to backup every keyspace at once.
```